### ISSUE TITLE
Unix Timestamp - wrong displaying 

### ISSUE THREAD

**Issue by denkhaus** [nocobase/nocobase#5961]
## Describe the bug

Since around version v1.4.14 I experience a displaying bug with unix timestamp fields. Despite the fact, that the field is defined and filled correctly the displayed value is wrong

## Screenshots
<details>
<summary>column representation</summary>
![column2024-12-27 113502](https://github.com/user-attachments/assets/d3095e67-1208-47bb-a811-81acf3dde596)
</details>

 and

<details>
<summary>underlying data</summary>
![data2024-12-27 113346](https://github.com/user-attachments/assets/a4f05e62-a2ec-4b71-b9c5-3fc2e846d460)
</details>

## Environment

- NocoBase version: v1.4.14
- Database type and version: PostgreSQL 16
- OS: linux
- Deployment Methods: fluxcd on kubernetes
- Docker image version: nocobase/nocobase:1.4.14

## How To Reproduce
Create a collection with timestamp field, filll in some data, add the respective column to the table...

## Expected behavior
I expect to see a valid DateTime value representing the unix timestamp field.

**Comment by mytharcher** [comment]
Hi @denkhaus , did you try v1.4.16? Which we just made a fix in (#5931). You can try it on new version to see if it works as your expected.

**Comment by denkhaus** [comment]
Yes, I'm now on version ``v1.4.16`` but the error stays the same.

**Comment by mytharcher** [comment]
Fixed on latest main.
